modules/godbcbackend/sodbc.cc

```diff
@@ -130,7 +132,8 @@ class SODBCStatement: public SSqlStatement
     p.ParameterValuePtr = (char*) new char[value.size()+1];
     value.copy((char*)p.ParameterValuePtr, value.size());
     ((char*)p.ParameterValuePtr)[value.size()]=0;
-
+    p.Ind = new SQLLEN;
+    *(p.Ind) = value.size();
```

Comment:
I could change the commit message, not sure.. feels bit wasteful not to use the Ind for rest since it needs to be allocated anyways.